config/default.yml

```diff
@@ -12,16 +12,43 @@ Lint/NestedMethodDefinition:
   Exclude:
     - api/sinatra/**/*
 
+Lint/RaiseException:
+  Enabled: true
+
+Lint/StructNewOverride:
+  Enabled: true
+
 ### Metrics
 
-Layout/LineLength:
+Metrics/AbcSize:
+  Max: 18
+  Severity: refactor
+  Exclude:
+    - api/sinatra/**/*
```

Comment:
this is not fitting our shed structure. Whats the intention of this cop with the severity level? not sure from the docs what it does

Reply:
Good question. This cop was just moved up to have some kind of order in here (See commit message).

I'll update the description.

Reply:
Should we even assume any folder structure here?

Reply:
we can update this one with the shed structure. intention was just to not have rubocop complain about about the "complexity" of the big blocks in the api files

Reply:
ah cool did not pay attention to the fact that this code only moved sorry, but maybe we can add the shed path for this as well, then it should be fine :)